src/views/v2/day/event/title.php

```diff
@@ -16,7 +16,7 @@
  * @see tribe_get_event() For the format of the event object.
  */
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: ce55233b78025feaa4487bd47869e626</sub>